src/haddock/clis/cli_bm.py

```diff
@@ -217,29 +216,23 @@ def job_setup(header, path, conf_f):
     header : str
         Configures queue manager system.
 
-    path : Path or str
+    run_path : Path or str
         Path where results will be stored. Must be synchronized with the
         `run_dir` path in the configuration file.
 
     conf_f : Path or str
         Path to the configuration file.
     """
+    conda_sh = Path(Path(sys.executable).parent,
```

Comment:
I would suggest we clean this line. what was the problem with `$HOME/miniconda3`?

Reply:
my miniconda3 is not in the $HOME :P

Reply:
Solved